megamek/src/megamek/client/ui/swing/RandomMapPanelAdvanced.java

```diff
@@ -1048,14 +1049,14 @@ private boolean isFieldVerified(VerifiableTextField field) {
         String result = field.verifyTextS();
         if (result != null) {
             result = field.getName() + ": " + result;
-            new RuntimeException(result).printStackTrace();
+            LogManager.getLogger().error("", new RuntimeException(result));
```

Comment:
Not sure why we're bothering to initialize a new exception here when just logging the string contents of it will do.

Reply:
This is done to log the full stacktrace as part of the error. It should be logging the results as the error message instead of blank though, which I'll fix.